src/render/draw_collision_debug.js

```diff
@@ -8,28 +8,37 @@ import type SymbolBucket from '../data/bucket/symbol_bucket';
 import DepthMode from '../gl/depth_mode';
 import StencilMode from '../gl/stencil_mode';
 import CullFaceMode from '../gl/cull_face_mode';
-import {collisionUniformValues} from './program/collision_program';
+import {collisionUniformValues, collisionCircleUniformValues} from './program/collision_program';
+
+import {StructArrayLayout2i4, StructArrayLayout3ui6, CollisionCircleLayoutArray} from '../data/array_types';
```

Comment:
Same as previous comment, I'd add names for these as well, even if they are just `QuadVerticesArray`